new-site/src/components/examples/FocusExample.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FocusExample = ({ color, background, style = {}, children }) => (
```

Comment:
Should we add the aria-labels and roles to this FocusExample element and for the ColourExample element (if missing)?
https://helsinkisolutionoffice.atlassian.net/wiki/spaces/DD/pages/7809761295/25.2.2022+saavutettavuustunti

Reply:
- [x] add missing aria-labels and role